drivers/scheduler/k8s/specs/nginx-simplified/px-nginx-app.yaml

```diff
@@ -0,0 +1,39 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: nginx-service
+spec:
+  selector:
+    app: nginx
+  type: NodePort
```

Comment:
Change `type: NodePort` to `type: ClusterIP`

Reply:
Fixed.